Cargo.lock

```diff
@@ -1748,7 +1748,7 @@ dependencies = [
 [[package]]
 name = "jsonrpc-core"
 version = "9.0.0"
-source = "git+https://github.com/paritytech/jsonrpc.git?branch=parity-2.2#f8a54f46f7f1d68b4e7899ca1e929803bf966a5b"
+source = "git+https://github.com/paritytech/jsonrpc.git?branch=parity-2.2#84360bf1d03821b8fd669a6a95fcdf847aaf6cca"
```

Comment:
why not a new branch called `parity-2.3` in the `jsonrpc` crate?

Reply:
There are no significant changes, it's pretty much just a bug fix. We usually create a new branch whenever there are some breaking changes.

Reply:
Ok, got it :+1: